lib/msal-core/src/cache/AuthCache.ts

```diff
@@ -126,6 +117,20 @@ export class AuthCache extends BrowserStorage {// Singleton
         }
     }
 
+    /**
+     * Reset all temporary cache items
+     */
+    resetTempCacheItems(): void {
```

Comment:
I think this function is supposed to be what removeAcquireTokenEntries does. I'd prefer not to have multiple functions doing similar jobs, so let's try to reconcile those differences and come up with a better strategy for cleaning up temporary keys.

Reply:
Done, Replaced removeAcquireTokenEntries with resetTempCacheItems now in the code.